pages/characters/index.tsx

```diff
@@ -36,22 +54,33 @@ const CharactersPage = () => {
 			</form>
 			<div className='flex flex-wrap justify-center w-full px-4 bg-transparent md:px-0 lg:mt-20 md:mt-16'>
 				{students.map((student: Student) => (
-					<div key={student.id} className='bg-transparent w-4/12 md:my-0 my-2'>
+					<div
+						key={student.username}
+						className='bg-transparent w-4/12 md:my-0 my-2'>
 						{isMobile ? (
 							<YearbookMobileProfileCard
-								key={student.id}
-								name={student.name}
-								image={student.image}
-								major={student.major}
-								house={student.house}
+								is_2021={student.is_2021}
```

Comment:
I think this data has no use for this feature, could you provide anything that will likely use it in the future?

Reply:
not sure yet. For now I'll just remove it